.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules
```

Comment:
From https://git-scm.com/docs/gitignore

```If the pattern ends with a slash, it is removed for the purpose of the following description, but it would only find a match with a directory. In other words, foo/ will match a directory foo and paths underneath it, but will not match a regular file or a symbolic link foo (this is consistent with the way how pathspec works in general in Git).```

Based on that I would suggest keeping the `/` at the end.